test/e2e/framework/pod/exec_util.go

```diff
@@ -129,6 +129,19 @@ func execCommandInPodWithFullOutput(ctx context.Context, f *framework.Framework,
 	return ExecCommandInContainerWithFullOutput(f, podName, pod.Spec.Containers[0].Name, cmd...)
 }
 
+func ExecCommandInContainerWithFullOutputUsingPodClient(ctx context.Context, f *framework.Framework, podName string, containerName string, cmd ...string) (string, string, error) {
```

Comment:
This is used exactly once. I don't think this should be added to `e2epod`.

You can put this code directly into your test.

Reply:
Thanks, will do that.